src/runtime/lpython_builtin.py

```diff
@@ -104,27 +104,28 @@ def str(x: i32) -> str:
     """
     Return the string representation of an integer `x`.
     """
-    if x == 0:
-        return '0'
-    result: str
-    result = ''
-    if x < 0:
-        result += '-'
-        x = -x
-    rev_result: str
-    rev_result = ''
-    rev_result_len: i32
-    rev_result_len = 0
-    pos_to_str: list[str]
-    pos_to_str = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
-    while x > 0:
-        rev_result += pos_to_str[x - _lpython_floordiv(x, 10)*10]
```

Comment:
Had to comment out this implementation, otherwise, it gives a type mismatch error (character and list).  We should fix this soon.